src/GLUtils.jl

```diff
@@ -38,7 +38,7 @@ end
 function mapkeys(func, collection::Dict)
     Dict([(func(key), value) for (key, value) in collection])
 end
-Base.get{KT, VT}(a::Dict{KT, VT}, keys::Vector{KT}) = [a[key] for key in keys]
+Base.get(a::Dict{KT, VT}, keys::Vector{KT}) where {KT, VT} = [a[key] for key in keys]
```

Comment:
@SimonDanisch can you use dot syntax here instead of extending base types on base methods?